src/index.js

```diff
@@ -74,14 +74,22 @@ module.exports = async function App(config) {
     )}`,
   );
 
-  await sendDelayedMessages(blocks => releaseCommunication.sendMessage(blocks), preparedBlocks);
+  try {
+    await sendDelayedMessages(blocks => releaseCommunication.sendMessage(blocks), preparedBlocks);
+  } catch (error) {
+    console.log(`Failed to send message to release channel ${error}`);
```

Comment:
ditto https://github.com/target/captains-log/pull/99/files#r588806463
```suggestion
    console.error('Failed to send message to release channel', error);
```